parser/parser.go

```diff
@@ -208,6 +208,134 @@ func parseUnary(p *Parser) (ast.Node, error) {
 	return ast.NewUnaryExpr(tok.Type, expr), nil
 }
 
+func parseIdentExpr(p *Parser) (ast.Node, error) {
+	tok := p.lookahead[0]
+	p.scry(1)
+	next := p.lookahead[1]
+
+	// eg.: console.
+	if next.Type == token.Dot {
```

Comment:
perhaps use something like "isMemberAccess(next)" to avoid the comment trying to explain what is happening

Reply:
function call as documentation?

Reply:
yup =)